### PR TITLE
fix crash QueryRenderedFeaturesBoxCountActivity

### LINKED ISSUE #822: QueryRenderedFeaturesBoxCountActivity crashing
**Describe the bug**
When launching QueryRenderedFeaturesBoxCountActivity, the test app crashes.

**To Reproduce**
1. Launch test app
2. Open the first activity in the "Features" catagory (which is QueryRenderedFeaturesBoxCountActivity)
3. App crashes

**Expected behavior**
Show toast message with feature count.

**Additional context**
```
    kotlin.UninitializedPropertyAccessException: lateinit property mapboxMap has not been initialized
        at com.mapbox.mapboxsdk.testapp.activity.feature.QueryRenderedFeaturesBoxCountActivity.getMapboxMap(QueryRenderedFeaturesBoxCountActivity.kt:23)
        at com.mapbox.mapboxsdk.testapp.activity.feature.QueryRenderedFeaturesBoxCountActivity.onStart(QueryRenderedFeaturesBoxCountActivity.kt:92)
```

### PR DESCRIPTION
**Pull request by louwers**
Closes #822

### CODE REVIEW

**Review by sulewicz — APPROVED** [review]

File: platform/android/MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/activity/feature/QueryRenderedFeaturesBoxCountActivity.kt [hunk 1]
```diff
@@ -89,7 +89,8 @@ class QueryRenderedFeaturesBoxCountActivity : AppCompatActivity() {
     override fun onStart() {
         super.onStart()
         mapView.onStart()
-        if (mapboxMap != null) {
+
```
Comment (ovivoda) [inline platform/android/MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/activity/feature/QueryRenderedFeaturesBoxCountActivity.kt:92]: @louwers I suggest checking both: mapboxMap != null && mapboxMap.isInitialized

Comment (louwers) [reply platform/android/MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/activity/feature/QueryRenderedFeaturesBoxCountActivity.kt:92]: @ovivoda Kotlin is null safe right?. It is not nullable so it cannot be null.

Comment (sulewicz) [reply platform/android/MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/activity/feature/QueryRenderedFeaturesBoxCountActivity.kt:92]: @louwers yes :)

Comment (ovivoda) [reply platform/android/MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/activity/feature/QueryRenderedFeaturesBoxCountActivity.kt:92]: https://bignerdranch.com/blog/kotlin-when-to-use-lazy-or-lateinit/

**Pull request merged**